testsuite/support.py

```diff
@@ -168,6 +168,12 @@ def init_tests(pep8style):
 
     def run_tests(filename):
         """Run all the tests from a file."""
+        # Skip tests meant for higher versions of python
+        ver_match = re.search(r'python(\d)(\d)?\.py$', filename)
+        if ver_match:
+            test_against_version = tuple(int(val or 0) for val in ver_match.groups())
```

Comment:
I probably gave you a poor suggestion here because of 

```
testsuite/support.py:174:80: E501:174:80 line too long (85 > 79 characters)
```

Perhaps

```suggestion
test_against_version = tuple(int(val or 0)
                             for val in ver_match.groups())
```

Reply:
It's my bad for not running a formatter before committing! maybe make an issue for pre-commit hooks for the repo, or add auto-formatting to the travis build. - so this doean't need to matter